2009-02-13  Michael Tautschnig  <mt@debian.org>

	* bin/fai-mirror: Fixed mess of $qflag, $quiet, $verbose and $debug
		(closes: #494431)
Index: trunk/bin/fai-mirror
===================================================================
--- trunk.orig/bin/fai-mirror
+++ trunk/bin/fai-mirror	
@@ -193,7 +193,7 @@
     echo "Adding packages from base.tgz."
     if [ -f $NFSROOT/var/tmp/base-pkgs.lis ]; then
 	plist=$(< $NFSROOT/var/tmp/base-pkgs.lis)
-	apt-get $quiet $aptoptions update >/dev/null
+	apt-get $qflag $aptoptions update >/dev/null
 	apt-get $qflag -d $aptoptions -y --force-yes --fix-missing install $plist
     else
 	echo "WARNING: $NFSROOT/var/tmp/base-pkgs.lis does not exists."
@@ -249,7 +249,7 @@
     echo "$cfdir is not a directory"
     exit 6
 fi
-[ "$verbose" ] && echo "Using configuration files from $cfdir"
+[ "$verbose" -eq 1 ] && echo "Using configuration files from $cfdir"
 . $cfdir/fai.conf
 . $cfdir/make-fai-nfsroot.conf
 export NFSROOT="$NFSROOT/live/filesystem.dir"
@@ -258,8 +258,6 @@
 [ -n "$NFSROOT_PACKAGES" ] && die "WARNING: The use of \$packages in make-fai-nfsroot.conf is now deprecated. Please include this information into $cfdir/NFSROOT."
 
 [ -n "$exclasses" -a -n "$cclasses" ] && die 3 "Options -x and -c not allowed at the same time."
-debug=0
-[ $debug -eq 0 ] && quiet=-q
 
 mirrordir=$1
 if [ -z "$mirrordir" ]; then
@@ -291,7 +289,7 @@
 perl -p -e 's/file:/copy:/' $cfdir/apt/sources.list > $aptcache/etc/apt/sources.list
 
 echo "Getting package information"
-apt-get $quiet $aptoptions update >/dev/null
+apt-get $qflag $aptoptions update >/dev/null
 
 set-classes
 echo "Downloading packages for classes:" $classes
@@ -314,7 +312,7 @@
 CONTENTS=no
 PKGCOMP='none gzip'
 EOF
-apt-move $quiet -c $aptmovefile update
+apt-move $qflag -c $aptmovefile update
 # since Packages.gz from apt-move does not include packages from my
 # repository, let's use apt-ftparchive for generiating correct index
 # files
